### PR TITLE
Updates for research group Wick

### PR DESCRIPTION
**Pull request by jpthiele**
These are new or missing publications of the research group of @tommeswick .

### CODE REVIEW

**Review by peterrum — COMMENTED** [review]
Very nice! Would you mind to make the bibtex entries a bit more consistent?

**Comment by jpthiele** [comment]
Now all additions are consistent and all checks pass. 
I was not sure whether I should make all the auto assigned bibtex identifiers consistent as well.
If yes, then I'll happily do that too.

**Review by peterrum — APPROVED** [review]
Good to go once you have resolved the merge conflicts.

File: publications-2021.bib [hunk 1]
```diff
@@ -668,4 +668,99 @@ @article{maier21c
   pages = {20210609},
 }
 
+
+@unpublished{160_PREPRINT,
```
Comment (marcfehling) [inline publications-2021.bib:672]: Is it still a preprint? I found this article here: https://link.springer.com/chapter/10.1007/978-3-030-87312-7_24

Could you add the doi and update the journal etc.?

File: publications-2021.bib [hunk 2]
```diff
@@ -668,4 +668,99 @@ @article{maier21c
...
+}
+
+@article{https://doi.org/10.1002/pamm.202100151,
+  author = {Beuchler, Sven and Endtmayer, Bernhard and Wick, Thomas},
+  title = {Goal oriented error control for stationary incompressible flow coupled to a heat equation},
+  journal = {PAMM},
+  volume = {21},
+  number = {1},
+  pages = {e202100151},
+  doi = {https://doi.org/10.1002/pamm.202100151},
```
Comment (marcfehling) [inline publications-2021.bib:687]:
```suggestion
  doi = {10.1002/pamm.202100151},
```

File: publications-2021.bib [hunk 3]
```diff
@@ -668,4 +668,99 @@ @article{maier21c
...
+}
+
+@article{https://doi.org/10.1002/pamm.202100175,
+  author = {Beuchler, Sven and Kinnewig, Sebastian and Koenig, Philipp and Wick, Thomas},
+  title = {A residual-based error estimator and mesh adaptivity for the time harmonic Maxwell equations applied to a Y-beam splitter},
+  journal = {PAMM},
+  volume = {21},
+  number = {1},
+  pages = {e202100175},
+  doi = {https://doi.org/10.1002/pamm.202100175},
```
Comment (marcfehling) [inline publications-2021.bib:701]:
```suggestion
  doi = {10.1002/pamm.202100175},
```

File: publications-2021.bib [hunk 4]
```diff
@@ -668,4 +668,99 @@ @article{maier21c
...
+}
+
+@article{https://doi.org/10.1002/pamm.202100174,
+  author = {Thiele, Jan Philipp and Wick, Thomas},
+  title = {Space-time PU-DWR error control and adaptivity for the heat equation},
+  journal = {PAMM},
+  volume = {21},
+  number = {1},
+  pages = {e202100174},
+  doi = {https://doi.org/10.1002/pamm.202100174},
```
Comment (marcfehling) [inline publications-2021.bib:715]:
```suggestion
  doi = {10.1002/pamm.202100174},
```

File: publications-2021.bib [hunk 5]
```diff
@@ -668,4 +668,99 @@ @article{maier21c
...
+  url = {https://onlinelibrary.wiley.com/doi/abs/10.1002/pamm.202100174},
+  eprint = {https://onlinelibrary.wiley.com/doi/pdf/10.1002/pamm.202100174},
+  abstract = {Abstract In this work, space-time goal-oriented a posteriori error estimation using a partition-of-unity localization is applied to the linear heat equation. The algorithmic developments are substantiated with a numerical example.},
+  year = {2021}
+}
+
+@InProceedings{JoLaWiDD26,
+  author= {D. Jodlbauer, U. Langer, T. Wick}, 
+  title={Efficient monolithic solvers for fluid-structure interaction applied to flapping membranes.},
```
Comment (marcfehling) [inline publications-2021.bib:724]:
```suggestion
  title={Efficient monolithic solvers for fluid-structure interaction applied to flapping membranes},
```

File: publications-2021.bib [hunk 6]
```diff
@@ -668,4 +668,99 @@ @article{maier21c
...
+  booktitle={Domain Decomposition Methods in Science and Engineering XXVI},
+  year={2021},
+  publisher={Springer International Publishing},
+  address={Cham},
+  pages={accepted},
+}
+
+@InProceedings{BeKiWiDD26,
+  author={D. Jodlbauer, U. Langer, T. Wick},
+  title={Parallel domain decomposition solvers for the time harmonic Maxwell equations.},
```
Comment (marcfehling) [inline publications-2021.bib:734]:
```suggestion
  title={Parallel domain decomposition solvers for the time harmonic Maxwell equations},
```

File: publications-2021.bib [hunk 7]
```diff
@@ -668,4 +668,99 @@ @article{maier21c
...
+  booktitle={Domain Decomposition Methods in Science and Engineering XXVI},
+  year={2021},
+  publisher={Springer International Publishing},
+  address={Cham},
+  pages={accepted},
+}
+
+@InProceedings{Wick2021EYIC,
+  author={T. Wick},
+  title={Adjoint-based methods for optimization and goal-oriented error control applied to fluid-structure interaction: implementation of a partition-of-unity dual-weighted residual estimator for stationary forward FSI problems in deal.II.},
```
Comment (marcfehling) [inline publications-2021.bib:744]:
```suggestion
  title={Adjoint-based methods for optimization and goal-oriented error control applied to fluid-structure interaction: implementation of a partition-of-unity dual-weighted residual estimator for stationary forward FSI problems in deal.II},
```

File: publications-2021.bib [hunk 8]
```diff
@@ -668,4 +668,99 @@ @article{maier21c
...
+  booktitle={Book of VI ECCOMAS Young Investigators Conference YIC2021},
+  year={2021},
+  url = {https://doi.org/10.4995/YIC2021.2021.12332},
```
Comment (marcfehling) [inline publications-2021.bib:747]:
```suggestion
  url = {10.4995/YIC2021.2021.12332},
```

File: publications-2021.bib [hunk 9]
```diff
@@ -668,4 +668,99 @@ @article{maier21c
...
+
+
+@article{NOII2021108060,
+  title = {A quasi-monolithic phase-field description for orthotropic anisotropic fracture with adaptive mesh refinement and primal-dual active set method},
+  journal = {Engineering Fracture Mechanics},
+  volume = {258},
+  pages = {108060},
+  year = {2021},
+  issn = {0013-7944},
+  doi = {https://doi.org/10.1016/j.engfracmech.2021.108060},
```
Comment (marcfehling) [inline publications-2021.bib:760]:
```suggestion
  doi = {10.1016/j.engfracmech.2021.108060},
```

File: publications-2022.bib [hunk 10]
```diff
@@ -83,4 +83,68 @@ @article{orlando:2022
...
+
+@article{AhEndtSteiWi22,
+  author = {K. Ahuja and B. Endtmayer and M.C. Steinbach and T. Wick},
+  title = {Multigoal-oriented error estimation and mesh adaptivity for fluid-structure interaction},
+  journal = {Journal of Computational and Applied Mathematics},
+  volume = {412},
+  pages = {114315},
+  year = {2022},
+  issn = {0377-0427},
+  doi = {https://doi.org/10.1016/j.cam.2022.114315},
```
Comment (marcfehling) [inline publications-2022.bib:122]:
```suggestion
  doi = {10.1016/j.cam.2022.114315},
```

File: publications-2022.bib [hunk 11]
```diff
@@ -83,4 +83,68 @@ @article{orlando:2022
...
+  title = {Computational performance studies for space-time phase-field fracture optimal control problems},
+  doi = {10.48550/ARXIV.2203.14643},
+  url = {https://arxiv.org/abs/2203.14643},
+  keywords = {Optimization and Control (math.OC), Numerical Analysis (math.NA), FOS: Mathematics, FOS: Mathematics, 74R10, 65N30, 49M15, 49K20, 35Q74},
+  publisher = {arXiv},
+  year = {2022},
+  copyright = {Creative Commons Attribution Non Commercial No Derivatives 4.0 International}
+}
+
+@article{diehl2022comparative,
```
Comment (marcfehling) [inline publications-2022.bib:105]: Can you update the entry according to https://link.springer.com/article/10.1007/s00466-022-02147-0

File: publications-2022.bib [hunk 12]
```diff
@@ -83,4 +83,68 @@ @article{orlando:2022
          journal = {International Journal for Numerical Methods in Fluids}
 }
 
+@InProceedings{ThiWi2022ICOSAHOM,
+  author={J. P. Thiele, T. Wick},
+  title={Space-time error control using a partition-of-unity dual-weighted residual method applied to low mach number combustion.},
```
Comment (marcfehling) [inline publications-2022.bib:88]:
```suggestion
  title={Space-time error control using a partition-of-unity dual-weighted residual method applied to low mach number combustion},
```

File: publications-2020.bib [hunk 13]
```diff
@@ -1023,4 +1023,13 @@ @article{maier20f
   pages = {109871},
 }
 
+@mastersthesis{Khi20,
```
Comment (marcfehling) [inline publications-2020.bib:1026]: Not mandatory, but an URL would be nice if one exists. Can this thesis be found anywhere online?

**Review by marcfehling — CHANGES_REQUESTED** [review]

File: publications-2020.bib (hunk 13 above)
Comment (jpthiele) [reply publications-2020.bib:1026]: Sadly, no.

File: publications-2021.bib (hunk 1 above)
Comment (jpthiele) [reply publications-2021.bib]: Yes indeed, that is already published by now. I updated the entry

File: publications-2021.bib [hunk 14]
```diff
@@ -668,4 +668,106 @@ @article{maier21c
...
+  address={Cham},
+  pages={accepted},
+}
+
+@InProceedings{Wick2021EYIC,
+  author={T. Wick},
+  title={Adjoint-based methods for optimization and goal-oriented error control applied to fluid-structure interaction: implementation of a partition-of-unity dual-weighted residual estimator for stationary forward FSI problems in deal.II},
+  booktitle={Book of VI ECCOMAS Young Investigators Conference YIC2021},
+  year={2021},
+  url = {10.4995/YIC2021.2021.12332},
```
Comment (marcfehling) [inline publications-2021.bib:754]:
```suggestion
  doi = {10.4995/YIC2021.2021.12332},
```

**Review by marcfehling — APPROVED** [review]
Skimming through the PR again, I found one small remaining issue.

It will be ready for merge, once you rebased on the master branch. It would be great if you squashed the commits into one in the process.

**Comment by marcfehling** [comment]
I took the liberty and made the according changes directly on your branch.

**Pull request merged**